src/app/business/block/bucket-detail/bucket-detail.component.html

```diff
@@ -35,7 +36,7 @@
                         </ng-template>
                     </p-column>
                     <p-column field="size" header="Size" [style]="{'width': '120px'}"></p-column>
-                    <p-column field="Backend" header="Location" [style]="{'width': '120px'}" ></p-column>
+                    <p-column field="Location" header="Location" [style]="{'width': '120px'}" ></p-column>
```

Comment:
This change is already pushed in #224 .

Reply:
I have undone the changes. Please review them. and I have seen the record of your submission of PR, which is later than mine. If there is a similar situation next time, please check whether I have submitted it first, so as to avoid code conflicts caused by everyone submitting. Thank you

Reply:
@anvithks I have made the change about the list of objects, plz check it.